docs/source/lesson/prg/python_intro.rst

```diff
@@ -261,27 +121,55 @@ At the end of this lesson you will be able to:
 - have an overview of the standard library
 - install Python libraries using ``virtualenv``
 
+Python Installation
+=================================================================
 
-.. _python-resources:
+Python is easy to install and very good instructions for most
+platforms can be found on the python.org Web page. We will be using
+Python 2.7.12 but not Python 3.
```

Comment:
Shouldn't this be 2.7.13? I don't see a problem with .12, except that except that our message has been that we support .13. I don't anticipate technical issues for .12, but am concerned about messaging consistency.